locale/fi/LC_MESSAGES/django.po

```diff
@@ -259,79 +332,104 @@ msgstr "Etusivu"
 msgid "TOS import"
 msgstr "TOS-tuonti"
 
-#: metarecord/views/admin.py:37
+#: metarecord/tests/test_api.py:2040 metarecord/views/classification.py:163
+#: metarecord/views/function.py:239
+msgid "Invalid state change."
+msgstr "Virheellinen tilan vaihto."
+
+#: metarecord/views/admin.py:40
 #, python-format
 msgid "File \"%s\" was imported successfully!"
 msgstr "Tiedosto \"%s\" tuotu onnistuneesti!"
 
-#: metarecord/views/admin.py:39
+#: metarecord/views/admin.py:42
 #, python-format
 msgid "Error importing file \"%s\""
 msgstr "Tiedoston \"%s\" tuonti epäonnistui"
 
-#: metarecord/views/base.py:97 metarecord/views/base.py:129
-#: metarecord/views/base.py:130
+#: metarecord/views/base.py:109 metarecord/views/base.py:141
+#: metarecord/views/base.py:142
 msgid "This attribute is required."
 msgstr "Tämä attribuutti on pakollinen."
 
-#: metarecord/views/base.py:105
+#: metarecord/views/base.py:117
 msgid "This attribute isn't allowed."
 msgstr "Tämä attribuutti ei ole sallittu."
 
-#: metarecord/views/base.py:109
+#: metarecord/views/base.py:121
 msgid "This attribute does not allow multiple values."
 msgstr "Tälle attribuutille ei voi antaa kuin yhden arvon."
 
-#: metarecord/views/base.py:119
+#: metarecord/views/base.py:131
 msgid "Value must be a string."
 msgstr "Arvo täytyy olla merkkijono."
 
-#: metarecord/views/base.py:125
+#: metarecord/views/base.py:137
 msgid "Invalid value."
 msgstr "Epäkelpo arvo."
 
-#: metarecord/views/function.py:115
-msgid "\"valid_from\" cannot be after \"valid_to\"."
-msgstr "\"valid_from\" ei voi olla myöhemmin kuin \"valid_to\"."
-
-#: metarecord/views/function.py:120
-#, python-format
-msgid "Classification %s already has a function."
-msgstr "Luokalla %s on jo käsittelyprosessi."
+#: metarecord/views/bulk_update.py:66
+msgid "No permission to create bulk update"
+msgstr "Ei lupaa luoda massamuutosta"
 
-#: metarecord/views/function.py:124
-#, python-format
-msgid "Classification %s does not allow function creation."
-msgstr "Luokalle %s ei voi lisätä käsittelyprosessia."
+#: metarecord/views/bulk_update.py:77
+msgid "No permission to update bulk update"
+msgstr "Ei lupaa päivittää massamuutosta"
 
-#: metarecord/views/function.py:134
-msgid "No permission to create."
-msgstr "Ei lupaa luoda."
+#: metarecord/views/bulk_update.py:103
+msgid "No permission to delete bulk update"
+msgstr "Ei lupaa poistaa massamuutosta"
 
-#: metarecord/views/function.py:162
+#: metarecord/views/classification.py:143 metarecord/views/function.py:170
 msgid "\"state\", \"valid_from\" or \"valid_to\" required."
 msgstr ""
 "Joku kentistä \"state\", \"valid_from\" tai \"valid_to\" on pakollinen."
 
-#: metarecord/views/function.py:192
-msgid "No permission to edit."
-msgstr "Ei lupaa muokata."
+#: metarecord/views/classification.py:175 metarecord/views/function.py:251
+msgid "No permission for the state change."
+msgstr "Ei oikeutta tilan vaihtoon."
```

Comment:
Nitpick-level 5/5, but I would use `lupaa` instead of `oikeutta` because that seems to be used in other texts when referring to permissions.